### DISCUSSION TITLE
Spray unit

### DISCUSSION THREAD

**Discussion by gwarkelly** [geezacoleman/OpenWeedLocator#147] [Show and tell]
![1000017443](https://github.com/user-attachments/assets/33140d79-dc0c-4513-841a-76589b9ea867)

**Comment by gwarkelly** [comment]
Hose now in boom
![Uploading 20240830_152543~3.jpg…]()

**Comment by geezacoleman** [comment]
Awesome work! That's a really neat way of doing it. Depends what you're spraying but you'll just want to make sure the hosing/nozzles are rated for it and won't degrade.
